docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml

```diff
@@ -127,29 +132,58 @@ code: |
+      - Changing the actual outcome of the decision: Changing the actual outcome of the decision
+      - Changing arguments made in court: Changing arguments made in court
+    none of the above: False
+---
+id: needs help with something else
+question: |
+  Warning
 subquestion: |
-  **You can use this form to file a Motion for:**
-  
-    1. Reconsideration, if you are asking the Court to reverse or otherwise change the outcome of your Appeal.
-    
-    2. Modification, if you are asking that an inaccuracy in the decision be corrected but the outcome to remain the same.
-    
-    3. Reconsideration and Modification, if you are asking the Court for both as defined above.
-  
-  **This form is *not* used for:**
-    
-    * A real-world change of circumstances
-    
-    * Changing the actual outcome of any decision
-    
-    * Changing arguments made in court
-   
-  For more information on what you need to file a Motion for Reconsideration and/or Modification, view the [Appellate Procedure for Rule 27](https://www.mass.gov/rules-of-appellate-procedure/appellate-procedure-rule-27-motion-for-reconsideration-or-modification).
-   
-  This form cannot be directly emailed to the Supreme Judicial Court. For more information, contact the Supreme Judicial Court Clerk's office: (617) 557-1020.
+  You can still proceed with your Motion for Reconsideration and/or Modification, but it may not be allowed because you are submitting it more than 14 days after the Appeals Court decision.
+buttons:
+  - Keep going:
+      code: |
+        needs_help_with_something_else_continue_anyway = True  
+  - Restart: restart
+  - Exit: exit
+---
+#---
+#comment: |
+#  This question is used to introduce your interview. Please customize
+#id: Motion to Reconsider
+#continue button field: Motion_to_Reconsider_intro
+#question: |
+#  What you need to know
+#subquestion: |
+#  **You can use this form to file a Motion for:**
+#  
+#    1. Reconsideration, if you are asking the Court to reverse or otherwise change the outcome of your Appeal.
+#    
+#    2. Modification, if you are asking that an inaccuracy in the decision be corrected but the outcome to remain the same.
+#    
+#    3. Reconsideration and Modification, if you are asking the Court for both as defined above.
+#  
+#  **This form is *not* used for:**
+#    
+#    * A real-world change of circumstances
+#    
+#    * Changing the actual outcome of any decision
+#    
+#    * Changing arguments made in court
+#   
+#  For more information on what you need to file a Motion for Reconsideration and/or Modification, view the [Appellate Procedure for Rule 27](https://www.mass.gov/rules-of-appellate-procedure/appellate-procedure-rule-27-motion-for-reconsideration-or-modification).
+#   
+#  This form cannot be directly emailed to the Supreme Judicial Court. For more information, contact the Supreme Judicial Court Clerk's office: (617) 557-1020.
```

Comment:
Our class specifically did not like how this looked to the user on the initial screen so we moved it to the warning screen.